### PR TITLE
Update data and formatting for claims list api data

### PR DESCRIPTION
**Pull request by jbalboni**
Closes [25](https://github.com/department-of-veterans-affairs/sunsets-team/issues/25)

Also adds a loading spinner while the api call is in progress.

Data left:
- [x] Status description
- [x] Need files flag

screen shot 2016-10-07 at 3 45 18 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@cagedcrown and @austin-martinez, want to take a look?

**Comment by carlmjohnson** [comment]
Looks cool to me.

File: src/js/disability-benefits/components/ClaimsListItem.jsx [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+          ? <p><i className="fa fa-envelope"></i>We sent you a development letter</p>
+          : null}
+        {claim.attributes.decisionLetterSent
+          ? <p><i className="fa fa-envelope"></i>We sent you a decision letter</p>
+          : null}
+        {!claim.attributes.decisionLetterSent && claim.attributes.documentsNeeded
+          ? <p><i className="fa fa-exclamation-triangle"></i>We need files from you</p>
+          : null}
+      </div>
+      <p>Last update: {moment(claim.attributes.updatedAt).format('MMM M, YYYY')}</p>
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: I'm not sure which date we want to show here, the last time we updated data from EVSS or the last update to the claim. If it's the last time we synced data from EVSS, this is correct. If it's the last update to the claim, I think you'd use the most recent date from the `events_list`

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: Ah, I didn't notice that. The date I was using before was `phaseChangeDate`, which is the only other update type date I see.

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: I'm changing this to phaseChangeDate for now.

File: src/js/disability-benefits/containers/YourClaimsPage.jsx [hunk 2]
```diff
@@ -37,7 +38,7 @@ class YourClaimsPage extends React.Component {
     let content;
 
     if (loading) {
-      content = <div>Loading...</div>;
+      content = <Loading/>;
```
Comment (cagedcrown) [inline src/js/disability-benefits/containers/YourClaimsPage.jsx]: Good call using a loading/spinner here.

**Comment by cagedcrown** [comment]
Everything looks good to me 👍

**Pull request merged**